docs/how-to-guides/running-feast-in-production.md

```diff
@@ -141,6 +202,145 @@ prediction = model.predict(feature_vector)
+
+The Java based feature server can be deployed to Kubernetes cluster via Helm charts in a few simple steps:
+
+1. Install [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) and [helm 3](https://helm.sh/)
+2. Add the Feast Helm repository and download the latest charts:
+```
+helm repo add feast-charts https://feast-helm-charts.storage.googleapis.com
+helm repo update
+```
+3. Run Helm Install
+```
+helm install feast-release feast-charts/feast \
+    --set global.registry.path=s3://feast/registries/prod \
+    --set global.project=<project name>
+```
+
+This chart will deploy two services: `feature-server` and `transformation-service`.
+Both must have read access to the registry file on cloud storage. Both will keep a copy of the registry in their memory and periodically refresh it, so expect some delays in update propagation in exchange for better performance.
+
+#### Load balancing
+
+The next step would be to install an L7 Load Balancer (eg, [Envoy](https://www.envoyproxy.io/)) in front of the Java feature server.
+For seamless integration with Kubernetes (including services created by Feast Helm chart) we recommend using [Istio](https://istio.io/) as Envoy's orchestrator.
+
+## 5. Ingesting features from a stream source
+
+Recently Feast added functionality for [stream ingestion](../reference/alpha-stream-ingestion.md).
+Please note that this is still in an early phase and new incompatible changes may be introduced.
+
+### 5.1. Using Python SDK in your Apache Spark / Beam pipeline
+
+The default option to write features from a stream is to add Python SDK into your existing PySpark / Beam pipeline.
+Feast SDK provides writer implementation that can be called from `foreachBatch` stream writer in PySpark like this:
+
+```python
+store = FeatureStore(...)
+
+def feast_writer(spark_df):
+    pandas_df = spark_df.to_pandas()
+    store.write_to_online_store("driver_hourly_stats", pandas_df)
+
+streamingDF.writeStream.foreachBatch(feast_writer).start()
+```
+
+### 5.2. Push service *(still under development)*
+
+Alternatively, if you want to ingest features directly from a broker (eg, Kafka or Kinesis) you can use "push service", which will write to online store on push call.
+This service will expose an HTTP API or when deployed on Serverless platforms like AWS Lambda or Google Cloud Function,
+this service can be directly connected to Kinesis or PubSub.
+
+If you are using Kafka, [HTTP Sink](https://docs.confluent.io/kafka-connect-http/current/overview.html) could be utilized as middleware.
+In this case "push service" can be deployed on Kubernetes or as a Serverless function.
+
+## 6. Monitoring
+
+Feast Services can report their metrics to a StatsD-compatible collector. To activate this function, you'll need to provide a StatsD IP address and a port when deploying the helm chart (in future will be added to `feature_store.yaml`).
```

Comment:
```suggestion
Feast services can report their metrics to a StatsD-compatible collector. To activate this function, you'll need to provide a StatsD IP address and a port when deploying the helm chart (in future, this will be added to `feature_store.yaml`).
```